docs/source/checks/nlp/data_integrity/plot_property_label_correlation.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_nlp_property_label_correlation:
+
+Property Label Correlation
+**************************
+
+This notebook provides an overview for using and understanding the "Property Label Correlation" check.
+
+**Structure:**
+
+* `What Is The Purpose of the Check? <#what-is-the-purpose-of-the-check>`__
+* `Run the Check <#run-the-check>`__
+
+What Is The Purpose of the Check?
+=================================
+The check estimates for every :doc:`text property </user-guide/nlp/nlp_properties>`
+(such as text length, language etc.) its ability to predict the label by itself.
+
+This check can help find a potential bias in the dataset - the labels being strongly correlated with simple text
+properties such as percentage of special characters, sentiment, toxicity and more.
+
+This is a critical problem, sometimes referred to as shortcut learning, where the model is likely to learn this property
```

Comment:
Maybe start by mentioning shortcut learning? Better opener than "find a potential bias"

Reply:
I think it's problematic to start from this as the "bias" line is required to explain shortcut learning.